### PR TITLE
feat:added grid

### PR DESCRIPTION
**Pull request by deepak29-git**
1.Two item grid
2.Three item grid

https://user-images.githubusercontent.com/88363801/152961138-3c89811d-ced9-4c2e-a1fe-3641fd202295.mp4

### CODE REVIEW

**Comment by spraveenofficial** [comment]
Looks good !

File: component/Grid/grid.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
+.two-item{
+    display:grid;
+    grid-auto-flow: row;
+    grid-gap:20px;
```
Comment (devananth) [inline component/Grid/grid.css]: Can be changed to `rem`

File: component/Grid/grid.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    color:var(--white-color);
+    padding: 1rem;
+    justify-content:center;
+    display:flex;
+    align-items:center;
+}
+.three-item{
+    display:grid;
+    grid-auto-flow: row;
+    grid-gap:20px;
```
Comment (devananth) [inline component/Grid/grid.css]: I think so these 3 properties are repeating , you refactor it.

**Review by devananth — CHANGES_REQUESTED** [review]
Good work !

File: component/Grid/grid.css [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+@media only screen and (max-width:600px){
+    .two-item{
+        grid-template-columns: repeat(1,1fr);
+    }
+    .three-item{
+        grid-template-columns: repeat(2,1fr);
+    }
+}
```
Comment (hinanshisuthar) [inline component/Grid/grid.css:42]: Add a line at the EOF

File: component/Grid/grid.html [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+  sandbox="allow-scripts allow-same-origin">
+</iframe>
+        </div>
+        <hr class="mb-2">
+              <div class="links mb-2">
+        <a class="link-tab" href="/component/Snackbar/snackbar.html"><span class="arrow"><</span>Snackbar</a>
+    </div>
+  </main>
+</body>
+</html>
```
Comment (hinanshisuthar) [inline component/Grid/grid.html:93]: Here, too.

**Review by hinanshisuthar — COMMENTED** [review]
Everything else looks perfectly fine.

File: component/Grid/grid.css (hunk 3 above)
Comment (deepak29-git) [reply component/Grid/grid.css:42]: ok

File: component/Grid/grid.html (hunk 4 above)
Comment (deepak29-git) [reply component/Grid/grid.html:93]: ok

File: component/Grid/grid.css (hunk 1 above)
Comment (deepak29-git) [reply component/Grid/grid.css]: ok

**Pull request merged**